crates/editor_ui/src/lib.rs

```diff
@@ -953,6 +848,27 @@ pub fn simple_editor_setup(mut commands: Commands) {
         ..default()
     });
 
+    // grid
+    const GRID_RENDER_LAYER: Layer = 1;
```

Comment:
Maybe make this pub, call it `LAST_RENDER_LAYER`, define it as `RenderLayers::TOTAL_LAYERS as u8 - 1` and use it in the other places `RenderLayers::TOTAL_LAYERS as u8 - 1` that is used. This helps because every editor specific layer will be in the last render layer.

Reply:
That's good to know, thanks! Changes made.